### DISCUSSION TITLE
Customize the 404\Orphan Page

### DISCUSSION THREAD

**Discussion by thallam** [roam-garden/i#47] [Feature requests & Ideas]
At the moment the orphan page is fixed. I would like to be able to customize it. 

One possible way would be to allow us to have a specific page on our Roam Graph for this purpose. eg. [[Roam Garden Orphan Page]] is what is displayed when we hit an orphan page.

**Comment by Stvad** [comment]
Yes, it'd be nice to do! As a fun exercise you can replace the text with CSS for the moment https://stackoverflow.com/questions/7896402/how-can-i-replace-text-with-css 😛
